shell/browser/api/electron_api_session.cc

```diff
@@ -1198,6 +1198,10 @@ gin::Handle<Session> Session::FromPartition(v8::Isolate* isolate,
     std::string name = partition.substr(8);
     browser_context =
         ElectronBrowserContext::From(name, false, std::move(options));
+  } else if (base::StartsWith(partition, base::FilePath::kSeparators,
+                              base::CompareCase::SENSITIVE)) {
+    browser_context =
+        ElectronBrowserContext::From(partition, false, std::move(options));
```

Comment:
```suggestion
  } else if (base::FilePath::FromUTF8Unsafe(partition).IsAbsolute() ||
             base::FilePath::FromUTF8Unsafe(partition).IsNetwork()) {
    browser_context =
        ElectronBrowserContext::From(partition, true, std::move(options));
```

Reply:
Hello Deepak, since the change now has a new fromPath API, do let me know if the IsNetwork() option is still needed.